### PR TITLE
Add unsigned 64 bit int read/write methods to cythonized stream

### PR DESCRIPTION
**Pull request by vikkyrk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @dhalperi @aaltay was able to verify locally that tests pass on my virtual env with cython enabled.

**Comment by vikkyrk** [comment]
Retest this please

File: sdks/python/apache_beam/coders/stream_test.py [hunk 1]
```diff
@@ -106,6 +106,15 @@ def test_read_write_bigendian_int64(self):
     for v in values:
       self.assertEquals(v, in_s.read_bigendian_int64())
 
+  def test_read_write_bigendian_uint64(self):
```
Comment (aaltay) [inline sdks/python/apache_beam/coders/stream_test.py]: Could you also update `test_byte_counting`

File: sdks/python/apache_beam/coders/stream.pyx [hunk 2]
```diff
@@ -63,7 +63,9 @@ cdef class OutputStream(object):
         break
 
   cpdef write_bigendian_int64(self, libc.stdint.int64_t signed_v):
-    cdef libc.stdint.uint64_t v = signed_v
+    self.write_bigendian_uint64(signed_v)
```
Comment (aaltay) [inline sdks/python/apache_beam/coders/stream.pyx]: Do you need the type casts similar to `read_bigendian_double` and `write_bigendian_double'?

Comment (vikkyrk) [inline sdks/python/apache_beam/coders/stream.pyx]: It is implicit as the method signature for `write_bigendian_uint64` has `libc.stdint.uint64_t`.

**Comment by dhalperi** [comment]
LGTM. It was not obvious to me where this is different than what I tried, but if your tests pass LGTM :)

File: sdks/python/apache_beam/coders/stream_test.py (hunk 1 above)
Comment (vikkyrk) [inline sdks/python/apache_beam/coders/stream_test.py]: Done

**Comment by vikkyrk** [comment]
@dhalperi I think you tried with `cpdef write_bigendian_uint64(self, libc.stdint.int64_t v)`. Note that the function parameter is `int64_t` instead of `uint64_t`

**Comment by aaltay** [comment]
LGTM. Thank you.

**Pull request closed without merging**